test/test.tf

```diff
@@ -0,0 +1,84 @@
+variable "dns_zone" {
+  default = "Z2J3KVPABDNIL1"
+}
+
+provider "aws" {
+  version                 = "2.6.0"
+  region                  = "ap-southeast-2"
+  alias                   = "ap-southeast-2"
+  shared_credentials_file = "/aws/credentials"
+  profile                 = "aeternity"
+}
+
+provider "aws" {
+  version = "2.6.0"
+  region  = "us-east-1"
+  alias   = "us-east-1"
+}
+
+variable "vault_addr" {
+  description = "Vault server URL address"
+}
+
+variable "bootstrap_version" {
+  default = "master"
+}
+
+variable "domain_sfx" {
+  default = ".ops.aeternity.com"
+}
+
+variable "envid" {
+  description = "Unique test environment identifier to prevent collisions."
+}
+
+variable "env_domain" {
+  default = "test-tf-gateway"
+}
+
+module "aws_deploy-test-gw" {
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v1.0.0"
```

Comment:
this module master should integrate with `terraform-aws-aenode-deploy` master not 1.0.0